src/ngsa/.editorconfig

```diff
@@ -200,6 +200,9 @@ dotnet_naming_style.begins_with_i.capitalization = pascal_case
 # SA0001: XML comment analysis
 dotnet_diagnostic.SA0001.severity = none
 
+# CA1002: Do not expose generic lists
+dotnet_diagnostic.CA1002.severity = none
+
```

Comment:
For whatever this is suppressing, could we return System.Collections.Generic.IList<T> instead so that the warning won't trigger instead of suppressing?

Reply:
We could, but List<> is the fastest of the generic collections, so suppressing for perf seems acceptable.